tests/components/dynalite/test_config_flow.py

```diff
@@ -104,3 +106,65 @@ async def test_two_entries(hass):
         )
     assert result["type"] == "create_entry"
     assert result["result"].state == "loaded"
+
+
+async def test_user_flow(hass):
+    """Run a flow generated by the user."""
+    host = "1.2.3.4"
+    with patch(
+        "homeassistant.components.dynalite.bridge.DynaliteDevices.async_setup",
+        return_value=True,
+    ):
+        result = await hass.config_entries.flow.async_init(
+            dynalite.DOMAIN, context={"source": config_entries.SOURCE_USER},
+        )
+        await hass.async_block_till_done()
+        assert result["type"] == "form"
+        assert result["step_id"] == "init"
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input={"host": host}
+        )
+        assert result["type"] == "create_entry"
+        assert result["result"].state == "loaded"
```

Comment:
Maybe also check the the data is correct?

Reply:
done